### PR TITLE
Laravel 10 support

### PR DESCRIPTION
**Pull request by masterjus**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,13 +11,13 @@
     }
   ],
   "require": {
-    "php": "^8.0",
+    "php": "^8.0|8.1",
```
Comment (andrew-nuwber) [inline composer.json]: I guess `^8.0` will handle 8.1 and 8.2

File: composer.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
   },
   "require-dev": {
     "mockery/mockery": "^1.4",
-    "phpunit/phpunit": "^9.0",
+    "phpunit/phpunit": "^9.0|^10.0",
```
Comment (andrew-nuwber) [inline composer.json:29]: Do we need support of v9?

File: .github/workflows/testing.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['8.0', '8.1']
+        php-versions: ['8.1', '8.2']
     name: Testing on PHP ${{ matrix.php-versions }}
     steps:
       - uses: actions/checkout@v2
```
Comment (andrew-nuwber) [inline .github/workflows/testing.yml:13]:
```suggestion
      - uses: actions/checkout@v3
```

**Review by andrew-nuwber — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (masterjus) [reply composer.json:29]: actially, no

**Review by andrew-nuwber — APPROVED** [review]

**Review by mrdzen — APPROVED** [review]

**Pull request merged**